LandSandBoat/server
# Trust Shikaree Z Gambits and Related

* Implements trust spell script for Shikaree Z (commented with TODOs)
* Populate Shikaree Z entries for mob_skill_lists
* Adds Erase to Shikaree Z mob_spell_lists
* Adds mob_skills/Raiden Thrust with SC properties
* Updates mob_skills/Red Lotus Blade to have SC properties
* Adds mobskills/raiden_thrust script



**_I affirm:_**

## What does this pull request do?
Adds scripts and plumbing to support Shikaree Z Trust

## Steps to test these changes
Build an appropriate Test Character
Summon Shikaree Z
Fight Mobs, observe behavior matches Gambits
